packages/jupyterlab-preview/src/preview.tsx

```diff
@@ -13,6 +13,8 @@ import {
 
 import { INotebookModel } from '@jupyterlab/notebook';
 
+import { refreshIcon } from '@juperlab/ui-components';
```

Comment:
looks like a type here.
```suggestion
import { refreshIcon } from '@jupyterlab/ui-components';
```